src/reversion/middleware.py

```diff
@@ -29,9 +29,7 @@ def _close_revision(self, request):
 
     def process_response(self, request, response):
         """Closes the revision."""
-        # look to see if the session has been accessed before looking for user to stop Vary: Cookie
-        if hasattr(request, 'session') and request.session.accessed \
-                and hasattr(request, "user") and request.user is not None and request.user.is_authenticated() \
+        if hasattr(request, "user") and request.user is not None and request.user.is_authenticated() \
```

Comment:
Are we absolutely certain that this won't result in the session being accessed?

I was actually wondering about making the check simply inspect request.method, and set the user if it's a POST, PATCH or PUT request. It's simpler, and mutating methods shouldn't suffer from losing the cache.